src/components/TransactionsInterpreter/TransactionsInterpreter.ts

```diff
@@ -1,20 +1,19 @@
+import { mainnetEgldLabel } from 'constants/network';
 import {
-  ExtendedTransactionType,
   TokenArgumentType,
-  UITransactionType
-} from './helpers/types';
-import getTransactionMethod from './helpers/getTransactionMethod';
-import { getTransactionTokens } from './helpers/getTransactionTokens';
-import urlBuilder from './helpers/urlBuilder';
-import { getDenominatedValue } from './helpers/getDenominatedValue';
+  ServerTransactionType
+} from 'types/server-transactions';
+import { getDenominatedValue } from 'utils/operations/getDenominatedValue';
+import { isContract } from 'utils/smartContracts';
+import { getTokenFromData } from 'utils/transactions/getTokenFromData';
 import { getNetworkLink } from './helpers/getNetworkLink';
-import { getEgldLabel, getTokenFromData } from 'utils';
-import { isContract } from './helpers/isContract';
-import { getTransactionType } from './helpers/getTransactionType';
-import { NUMBER_OF_CHARACTERS_FOR_SMART_CONTRACT_ADDRESS } from 'constants/transaction-interpreter';
-import { parseTransactionTime } from './helpers/parseTransactionTime';
+import { getTransactionMethod } from './helpers/getTransactionMethod';
 import { getTransactionReceiver } from './helpers/getTransactionReceiver';
 import { getTransactionReceiverAssets } from './helpers/getTransactionReceiverAssets';
+import { getTransactionTokens } from './helpers/getTransactionTokens';
+import { getTransactionTransferType } from './helpers/getTransactionTransferType';
+import { ExtendedTransactionType } from './helpers/types';
+import urlBuilder from './helpers/urlBuilder';
```

Comment:
use named exports instead of default. 
search for all `export default` in project